sdk/search/azure-search-documents/azure/search/documents/_generated/_serialization.py

```diff
@@ -1502,7 +1502,7 @@ def _deserialize(self, target_obj, data):  # pylint: disable=inconsistent-return
 
         :param str target_obj: Target data type to deserialize to.
         :param object data: Object to deserialize.
-        :raises: DeserializationError if deserialization fails.
+        :raises SerializationError: if serialization fails.
```

Comment:
same here

Reply:
my comments are out of order, see https://github.com/Azure/azure-sdk-for-python/pull/38872#discussion_r1884564071